### PR TITLE
Switch to klog v2 [kubernetes-sigs/apiserver-network-proxy]

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:https://github.com/kubernetes/kubernetes/issues/89961

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,12 +13,11 @@ require (
 	github.com/prometheus/procfs v0.0.0-20190507164030-5867b95ac084 // indirect
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
 	google.golang.org/grpc v1.26.0
 	k8s.io/api v0.17.1
 	k8s.io/apimachinery v0.17.1
 	k8s.io/client-go v0.17.1
-	k8s.io/klog v1.0.0
+	k8s.io/klog/v2 v2.0.0-rc.1
```
Comment (serathius) [inline go.mod:20]: Please change to v2.0.0

Comment (tanjunchen) [reply go.mod]: sure

**Comment by dims** [comment]
@Jefftree yes the plan is to update all the deps (for k/k) to klog v2 and then bump all those together (see checklist in https://github.com/kubernetes/kubernetes/issues/89961)

**Comment by tanjunchen** [comment]
kindly ping /cc @Jefftree

**Comment by andrewsykim** [comment]
/assign @anfernee

as well :)

**Comment by Jefftree** [comment]
/lgtm

/cc @caesarxuchao 

as well

**Comment by dims** [comment]
@Jefftree please remove hold as well

**Pull request merged**